tools/integration_tests/util/setup/implicit_and_explicit_dir_setup/implicit_and_explicit_dir_setup.go

```diff
@@ -72,15 +77,55 @@ func RemoveAndCheckIfDirIsDeleted(dirPath string, dirName string, t *testing.T)
 	}
 }
 
-func CreateImplicitDirectoryStructure(testDir string) {
+type objectCreationMetadata struct{ content, completeObjectName string }
+
+func createObjectsOnGcs(objects []objectCreationMetadata, t *testing.T, sgClient *storage.Client) {
+	ctx := context.Background()
```

Comment:
Shouldn't we catch this context similar to StorageClient

Reply:
Not unless this variable is to be used elsewhere. As I just added it here now, it is not needed outside this function.
storageClient is used in other places (tests), so it needed to be saved as variable.

Reply:
But we have used context while create client. Not sure if we should use that context only.

Reply:
Reopening this..

Reply:
addressed.